src/main/kotlin/gg/essential/elementa/impl/InternalApi.kt

```diff
@@ -0,0 +1,4 @@
+package gg.essential.elementa.impl
+
+@InternalApi
+annotation class InternalApi
```

Comment:
Would this have better IDE integration if we were to use `@ApiStatus.Internal` from JB annotations instead?

Reply:
Indeed does.